draft-linus-trans-gossip-ct.md

```diff
@@ -594,7 +600,7 @@ taking on this role needs to consider the following:
   SCT and an STH any time after the log's MMD has elapsed from the
```

Comment:
I think I frequently say "inclusion proof between SCT and STH" because I got that mixed up.  Should be "between cert and STH"